PowerEditor/src/WinControls/ImageListSet/ImageListSet.cpp

```diff
@@ -50,6 +50,28 @@ void IconList::addIcon(int iconID) const
 	::DestroyIcon(hIcon);
 };
 
+bool IconList::addInvertIcon(int iconID) const
+{
+	HICON hIcon = ::LoadIcon(_hInst, MAKEINTRESOURCE(iconID));
+	if (!hIcon)
+		throw std::runtime_error("IconList::addIcon : LoadIcon() function return null");
+
+	HICON hColorInvertedIcon = invertColour(hIcon);
+
+	if (hColorInvertedIcon)
+		ImageList_AddIcon(_hImglst, hColorInvertedIcon);
+
+	::DestroyIcon(hIcon);
+
+	return hColorInvertedIcon != 0;
+}
+
+HICON IconList::invertColour(HICON /*hIcon*/) const

```

Comment:
@adzm 
I know it's quite naive, but is it possible to implement this function to invert the colour of an icon ?
If the answer is yes, then we don't need to add dark mode icons for dark mode.
Ref our conversation:
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/9742#issuecomment-826142399

Reply:
@adzm 
Never mind. After checking [the link](https://github.com/derceg/explorerplusplus/commit/839a3fb36a9084fa4def1e0c6113e86c0a170738) provided by @WildByDesign , I think we will putting a lot of codding effort for not so good quality of icons.